bugwarrior/config/schema.py

```diff
@@ -151,17 +179,14 @@ class Config(PydanticConfig):
     #: Interactive status.
     interactive: bool
 
-    # added during validation (computed field support will land in pydantic-2)
-    #: Local data storage.
-    data: typing.Optional[BugwarriorData] = None
+    # optional
+    taskrc: TaskrcPath = Field(default_factory=get_default_taskrc)
```

Comment:
Please move this back below the `data` property -- the `# optional comment` applies to that whole next block of attributes.